DirectProgramming/C++SYCL_FPGA/README.md

```diff
@@ -85,7 +85,7 @@ flowchart LR
 | [double_buffering](Tutorials/DesignPatterns/double_buffering)                                                                 | [Tutorials/DesignPatterns](Tutorials/DesignPatterns)                                     | How and when to implement the double buffering optimization technique
 | [explicit_data_movement](Tutorials/DesignPatterns/explicit_data_movement)                                                     | [Tutorials/DesignPatterns](Tutorials/DesignPatterns)                                     | How to explicitly manage the movement of data for the FPGA
 | [hostpipes (experimental)](Tutorials/Features/experimental/hostpipes)                                                         | [Tutorials/Features](Tutorials/Features)                                                 | How to use host pipes to send and receive data between a host and the FPGA 
-| [invocation_interfaces](Tutorials/Features/hls_flow_interfaces/invocation_interfaces/)                                    | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces/)                                 | Demonstrates how to specify the kernel invocation interface and kernel argument interface for an FPGA IP produced with the Intel® oneAPI DPC++/C++ Compiler. 
+| [invocation_interfaces](Tutorials/Features/hls_flow_interfaces/invocation_interfaces/)                                        | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces/)                                 | Demonstrates how to specify the kernel invocation interface and kernel argument interface for an FPGA IP produced with the Intel® oneAPI DPC++/C++ Compiler. 

```

Comment:
What is being changed here?

Reply:
adding spaces to align the table divider line "|"